reference/data/models/base.v1.yaml

```diff
@@ -6,7 +6,9 @@ properties:
   annotations:
     $ref: '../../common/models/annotationarray.v1.yaml'
   archivedTime:
-    $ref: '../../common/models/datetime.v1.yaml'
+    allOf:
```

Comment:
I guess this construct is needed because there's no other way to make the data model referenced by `$ref` to be `readOnly: true`?

Should we apply the same treatment to `archiveDatasetId` as well, then?

Reply:
Ah, I was just experimenting here to see if it would work. Seems like it does! I think we could apply this to everything that is read-only?